### PR TITLE
Visit dao introduced

### PR DESCRIPTION
**Pull request by bkompis**
This pull request contains commit from previous pull request

### CODE REVIEW

File: persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ *
+ * @author bkompis
+ */
+public interface VisitDao {
+
+    /**
+     * Create a visit.
+     * @param visit the Visit to create in the database.
+     */
+    void create(Visit visit);
```
Comment (Lindar84) [inline persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java:18]: nevadi, ze tam neni PUBLIC ...?

File: persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    List<Visit> findAll();
+
+    /**
+     * Searches for visits whose date is between 'from' and 'to'.
+     * @param from beginning of the date interval
+     * @param to end of the date interval
+     * @return list of Visits
+     */
+    List<Visit> findByDate(LocalDate from, LocalDate to);
+    // TODO: maybe add findByHunter and findByForest
```
Comment (Lindar84) [inline persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java:52]: "CRUD operations are enough for the first checkpoint" ....

File: persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java (hunk 1 above)
Comment (bkompis) [reply persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java:18]: Nevadi, interface metody su public by default. Let's keep the conversations in English, please.

File: persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java (hunk 2 above)
Comment (bkompis) [reply persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java:52]: Which is why the other searches are commented out. But yes, this is not necessary.

**Review by Buvko — APPROVED** [review]

**Pull request merged**